exercises/reforestation/canonical-data.json

```diff
@@ -0,0 +1,93 @@
+{
+    "exercise": "reforestation",
+    "version": "1.0.0",
+    "cases": [
+      {
+        "description": "Base case: empty tree",
+        "property": "treeTraversals",
+        "input": {
+          "preorder": [],
+          "inorder": []
+        },
+        "expected": {}
+      },
+      {
+        "description": "1 more step than base case",
```

Comment:
This description refers to a previous case. Perhaps it would be simpler if this would be named "Tree with single node" or something like that, without having to refer to another test case.

Reply:
Okay. Will rename. I have the feeling most exercises start of with only one test case enabled and let the exerciser enable them as they go. Personally I just activate all of them.